### PR TITLE
export dgp functions

### PR DESCRIPTION
**Pull request by n-kall**

### CODE REVIEW

**Review by yannmclatchie — APPROVED** [review]
lgtm!

**Comment by n-kall** [comment]
fixed some warnings and improved documentation

File: R/generate_from_dgp.R [hunk 1]
```diff
@@ -27,19 +28,30 @@ generate_from_dgp <- function(dgp, n_datasets, ...) {
```
Comment (TeemuSailynoja) [inline R/generate_from_dgp.R:28]: Should we assert that the values returned by the dgp are of a certain format, or is it better to leave this more general?

Comment (TeemuSailynoja) [reply R/generate_from_dgp.R:28]: Discussed with Noa and agreed that keeping things general is better here.

**Review by TeemuSailynoja — APPROVED** [review]

**Pull request merged**